ha/setup/ha_setup.py

```diff
@@ -139,6 +145,41 @@ def get_node_name(self):
         Log.info(f"Read node name: {node_name}")
         return node_name
 
+    def get_nodelist(self, fetch_from: str) -> list:
```

Comment:
fetch_from = { PROV_CONFSTORE, HA_CONFSTORE }
Also if some other string is passed then throw an exception.

Reply:
Agree

Reply:
Done